src/Amazon.ElasticBeanstalk.Tools/Commands/DeployPackageCommand.cs

```diff
@@ -0,0 +1,600 @@
+﻿using Amazon.Common.DotNetCli.Tools;
+using Amazon.Common.DotNetCli.Tools.Options;
+using Amazon.ElasticBeanstalk.Model;
+using Amazon.S3.Transfer;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using ThirdParty.Json.LitJson;
+
+namespace Amazon.ElasticBeanstalk.Tools.Commands
+{
+    public class DeployPackageCommand : EBBaseCommand
```

Comment:
Instead of creating a separate deploy command can we reuse the existing deploy-environment that has an optional package command? If that is set deploy-environment would skip the packaging park.

Reply:
might have gotten this working